website/blog/2025-04-20-vs-code-with-podman-desktop.md

```diff
@@ -0,0 +1,100 @@
+
+![enabling docker compatibility in the settings](img/vs-code-podman/docker-compatibility.png)
+
+### Option 2: Pod Manager
+
+To install the extension:
+
+1. Open VS Code.
+2. Click on the Extensions icon in the Activity Bar (or press `Ctrl+Shift+X` or `Cmd+Shift+X`).
+3. Search for "Pod Manager" and install the extension by dreamcatcher45.
+
+## Using the VS Code Extensions
+
+Now that we have the extensions installed and configured, let's see how to use them.
+
+### Working with Containerfiles and Dockerfiles
+
+Both extensions provide syntax highlighting, code completion, and linting for Containerfiles and Dockerfiles. Open a Containerfile in VS Code, and you'll immediately benefit from these features.
+
+You can also build images directly from VS Code:
+
+1. Right-click on the Containerfile in the Explorer view.
+2. Select "Build Image".
+3. VS Code will prompt you for an image name and tag.
+4. The extension will then build the image using Podman.
+
+![building a Containerfile in vs code using the microsoft extension](img/vs-code-podman/build-image.png)
+
+### Managing Containers
+
+The extensions also allow you to manage containers directly from VS Code. You can start, stop, restart, and remove containers, as well as view their logs and inspect their configuration.
+
+To view the container logs:
+
+1. Click on the Container icon in the Activity Bar.
+2. You'll see a list of your containers, images, and networks.
+3. Right-click on a container to perform actions.
+
+![using Container Tools extension to view the logs of the container](img/vs-code-podman/view-logs.png)
+
+Similarly using the Pod manager extension we can visually inspect containers, images, and volumes.
+
+1. Click on the Pod manager icon in the Activity Bar.
+2. You'll see a list of your containers, images, and networks.
+
+![using podmanager to view all the running containers, images and volumes](img/vs-code-podman/podmanager-details.png)
+
+and manage the container lifecycle.
+
+![using podmanager to manage the lifecycle of the container](img/vs-code-podman/interact-with-container.png)
+
+and of course troubleshoot issues with a visual interface.
+
+![using podmanager to enter the container](img/vs-code-podman/podmanager-details.png)
+
+## Conclusion
+
+By combining the power of VS Code, Podman, and Podman Desktop, you can create a streamlined and efficient container development workflow. The VS Code extensions provide excellent integration with Podman, allowing you to manage containers, build images, and work with Containerfiles directly from your code editor. Podman Desktop complements this with a visual interface for managing your container environment. Embrace these tools and elevate your container development experience!
```

Comment:
I think it could be nice to frame that with regards to the recent changes in the ecosystem and that we are really looking forward to what the future holds for developers.
Again, tapping on the change on the MSFT VSCode extension would be important here.